plugins/processors/t128_filter/README.md

```diff
@@ -0,0 +1,74 @@
+# T128 Filter Processor Plugin
+
+The `t128_filter` filters out metrics passing through it. This processor is useful if the sort of filtering you wish to accomplish which is not possible with `tagpass`/`tagdrop`. The most usual case this processor can help is if you have two tags that you want both to be a certain value before passing along the metric.
+
+[Metric selectors](docs/CONFIGURATION.md#selectors) (such as `tagpass` and `tagdrop`) can be used to only apply the filters to specific metrics.
+
+### Configuration:
+
+```toml
+[[processors.t128_filter]]
+  ## The conditions that must be met to pass a metric through. This is similar
+  ## behavior to a tagpass, but multiple tags are ANDed
+  [[processors.t128_filter.condition]]
```

Comment:
Can you add an example with multiple conditions and the fact that its a logical OR between condition blocks. That part is not clear right now. I think it helps to understand why the `condition` keyword exists at all in this case.

Reply:
There is an example below and there is context that says that multiple conditions are AND'd together by default (will be configurable in the future)